source/declarations.tex

```diff
@@ -1717,12 +1717,8 @@
 each of which shall
 be followed by a non-empty
 \grammarterm{initializer}.
-In an \grammarterm{initializer} of the form
-\begin{codeblock}
-( @\textrm{\grammarterm{expression-list}}@ )
-\end{codeblock}
-the \grammarterm{expression-list} shall be a single
-\grammarterm{assignment-expression}.
+If the \grammarterm{initializer} is a parenthesized \grammarterm{expression-list},
+the \grammarterm{expression-list} shall be an \grammarterm{assignment-expression}.
```

Comment:
We should keep "single" to highlight that the expression-list isn't really list-y.